airflow/executors/executor_loader.py

```diff
@@ -134,7 +134,7 @@ def import_executor_cls(cls, executor_name: str) -> Tuple[Type["BaseExecutor"],
 
     @classmethod
     def __load_celery_kubernetes_executor(cls) -> "BaseExecutor":
-        """:return: an instance of CeleryKubernetesExecutor"""
+        """:return: an instance of CeleryKubernetesExecutor."""
```

Comment:
These _return an instance of $Executor_ docstrings should simply be removed, they do not really communicate anything.